building windows wheels fails due to a change in vcpkg

- PyBaMM version - `$ python -m pip show pybamm`: 21.08
- Python version - `$ python --version`: n/a

**Describe the bug**

The `.github/workflows/build_wheels.yml` workflow fails when building the windows wheel, with the following error:

```
---------- Running CMake for idaklu solver ----------------------------------------
    -- Building for: Visual Studio 16 2019
    -- Running vcpkg install
    Error: sundials:x86-windows-static@5.7.0#1 does not have required feature klu
    
    -- Running vcpkg install - failed
    CMake Error at C:/vcpkg/scripts/buildsystems/vcpkg.cmake:827 (message):
      vcpkg install failed.  See logs for more information:
      C:\Users\runneradmin\AppData\Local\Temp\pip-req-build-_9f138ur\build\temp.win32-3.6\vcpkg-manifest-install.log
    Call Stack (most recent call first):
      C:/Program Files/CMake/share/cmake-3.21/Modules/CMakeDetermineSystem.cmake:124 (include)
      CMakeLists.txt:18 (project)
    
    
    -- Configuring incomplete, errors occurred!
```


**To Reproduce**

merge to main

**Expected behaviour**

pypi wheels are build and deployed to pypi

--- a/vcpkg-configuration.json
+++ b/vcpkg-configuration.json
@@ -1,10 +1,14 @@
 {
+  "default-registry": {
+    "kind": "builtin",
+    "baseline": "c26477297ce1a9d67844e86bf0cda0e7741bd169"
+  },
   "registries": [
     {
       "kind": "git",
       "repository": "https://github.com/pybamm-team/sundials-vcpkg-registry.git",
-      "baseline": "3cdbafc061c4cdda2377c77c86aeb72cfcfbada6",
-      "packages": [ "sundials" ]
+      "packages": [ "sundials" ],
+      "baseline": "c19f72020d323811c5a5adf96235b961ec4ef89a"
     }
   ]
 }

--- a/vcpkg-configuration.json
+++ b/vcpkg-configuration.json
@@ -1,7 +1,7 @@
 {
   "default-registry": {
     "kind": "builtin",
-    "baseline": "c26477297ce1a9d67844e86bf0cda0e7741bd169"
+    "baseline": "642d1ccb276ce13d8a8521b1ef2856f5f7bc9cdd"
   },
   "registries": [
     {

